sdk/eventhub/event-hubs/src/receiver.ts

```diff
@@ -73,7 +73,8 @@ export class EventHubConsumer {
   /**
    * @property The set of retry options to configure the receiveBatch operation.
    */
-  private _retryOptions: Required<Pick<RetryOptions, "maxRetries" | "retryInterval">>;
+  private _retryOptions: Required<Pick<RetryOptions, "maxRetries" | "retryInterval">> &
```

Comment:
Since we can set a default value for `retryPolicy`, it should also be a required field.